prow/cmd/deck/main.go

```diff
@@ -191,6 +191,7 @@ func traceHandler(h http.Handler) http.Handler {
 }
 
 func main() {
+	metrics.ExposeMetrics("deck", "", time.Nanosecond)
```

Comment:
You should be able to read the values you need from config, right?

```go
	// Expose prometheus metrics
	pushGateway := cfg().PushGateway
	metrics.ExposeMetrics("deck", pushGateway.Endpoint, pushGateway.Interval)
```

Reply:
done